src/terraform-provider-paperspace/client.go

```diff
@@ -334,3 +349,22 @@ func (paperspaceClient *PaperspaceClient) GetTeamNamedNetworkById(teamID int, id
 
 	return nil, fmt.Errorf("Error getting private network by id: %s", id)
 }
+
+func (paperspaceClient *PaperspaceClient) GetJobStorageByRegion(teamID int, region string) (JobStorage, error) {
```

Comment:
can this note take a team handle? :\ if so maybe colin would be willing to update this endpoint as well

Reply:
no, it doesn't. This is an accounts endpoint

Reply:
i see, so this is really more of an internal api. maybe we should split it out into a private provider